ietf/settings.py

```diff
@@ -685,6 +685,7 @@ def skip_unreadable_post(record):
 MEETING_RECORDINGS_DIR = '/a/www/audio'
 DERIVED_DIR = '/a/ietfdata/derived'
 FTP_DIR = '/a/ftp'
+ALLID_DOWNLOAD_DIR = '/a/www/www6s/download'
```

Comment:
Is `ALLID` a term in use already? If not, maybe `ALL_ID`? I struggle to parse the former.